tests/fuzz/csmith_driver.py

```diff
@@ -176,8 +176,8 @@ def try_js(args=[]):
   def execute_js(engine):
     print('(run in %s)' % engine)
     try:
-      js = jsrun.run_js(filename + '.js', engine=shutil.NODE_JS, check_timeout=True, assert_returncode=None)
-    except CalledProcessError:
+      js = shared.timeout_run(Popen(shared.NODE_JS + [filename + '.js'], stdout=PIPE, stderr=PIPE), 15 * 60)
```

Comment:
this is the only line I'm not totally sure about..

Reply:
The timeout here is probably larger than it needs to be, could be 5 seconds or so probaly.

Also worth running this to see if it works. However, I'm not sure any of us has run the csmith driver for a while, so maybe it broke meanwhile.

Reply:
Does it need timeout?  I guess its there in case the generated code doesn't terminate?

Reply:
Yes, csmith code may not terminate (unlike the binaryen fuzzer output, which guarantees it)